src/components/profile/VerifyEdit.js

```diff
@@ -24,13 +25,15 @@ const EditProfile = ({ screenProps, theme, styles, navigation }) => {
   const field = get(navigation, 'state.params.field')
   const content = get(navigation, 'state.params.content')
   let fieldToSend
+  let fieldToSave
```

Comment:
what is this for?

Reply:
the "field" param was "phone" the field it needs to save to is "mobile"

Reply:
its duplicated with fieldToSend

Reply:
they have the same value but they are 2 separate things

Reply:
if field unchanged - we don't need to save it. so just skip it when you process profile

Reply:
we are only sent an API request if the field was changed

Reply:
ok